doc/modules/mod_mam.md

```diff
@@ -31,6 +31,10 @@ For now `odbc` backend has very limited support for this feature, while `cassand
 * **archive_chat_markers** (boolean, default: `false`) - If set to true, XEP-0333 chat markers will be archived. See more details [here](#archiving-chat-markers)
 * **pm** (list | `false`, default: `[]`) - Override options for archivization of one-to-one messages. If the value of this option is `false`, one-to-one message archive is disabled.
 * **muc** (list | `false`, default: `false`) - Override options for archivization of group chat messages. If the value of this option is `false`, group chat message archive is disabled.
+* **extra_lookup_params** (atom, default: `undefined`) - a module implementing `mam_iq` behaviour.
+ If this option has value other then undefined, function `extra_lookup_params/2` from this module will be called when building MAM lookup parameters.
```

Comment:
When building MAM lookup parameters the function `extra_lookup_params/2` is called  for values other than 'undefined'?

Reply:
It's not clear to me in the way you suggested @goddammit. What I wanted to say is exactly the following.

If the value of `extra_lookup_params` is other then `undefined`, it means this is a `module`. The function `extra_lookup_params/2` from the `module` will be called when building MAM lookup parameters.

Hope this explains my intention and helps to build better sentence if needed :)